OpenSimRT/Vicon/include/ViconDataStream.h

```diff
@@ -32,7 +32,6 @@
 #include "CircularBuffer.h"
```

Comment:
Tsaopoulos mentioned that we could create a docker that can set up this environment so that it could be run on Windows. This might be nice because we can send him the docker file and test it without being there to build the whole system. We might have to include the lower limb model that was used for the experiments. Did you change it?

Reply:
We used the same model for the experiments.